BCH-Wunder-Project-team-4/Wunder.io
# Latest DB and fix small issues

## Link to feature environment:

http://localhost:3000/

## Changes proposed in this PR:

- Added the latest DB with content updates: 
  - Author picture
  - Speaker avatar
  - Design Forum 2024 connected to right event webform and one speaker added for demo purpose
  - Contact infos are more demo-like (using dummy emails and phone numbers)
  - Added Finnish translation for contact_greeting in Contact Us (Ota yhteyttä) page
- Fixed small issued I spotted (check commits)
- Small style updates in couple of paragraphs

## How to test:

1. First update the git remote origin as the repository name has been changed to WunderProject
- Use `git remote set-url origin https://github.com/BCH-Wunder-Project-team-4/WunderProject.git`
2. `git pull origin text-image-update`
3. `git checkout origin text-image-update`
4. Import database `lando db-import drupal10.2024-01-11-DBLatest.sql.gz`
5. You should now see author image in articles and speaker image in Design Forum 2024 event.

## Best practices:

<details>
<summary><h3>Accessibility:</h3></summary>
<p>
This project must support WCAG accessibility level AA <em>(edit this according to the requirements of your project)</em>. To ensure this standard is met, remember to:

- Perform automated checks using a tool such as Wave or SiteImprove.
- Test keyboard navigation: are all parts of the UI navigable using only the keyboard? Is the tab order logical? Can popups, menus etc be dismissed with the escape key?
- Test responsiveness, scaling and text reflow.
- Make sure no accessibility issues exist on either desktop or mobile views.
- If you have time, test with a screen reader such as VoiceOver (macOS), NVDA (Windows), or Orca (Linux).

Use the [Accessibility Testing Cheat Sheet](https://intra.wunder.io/info/accessibility-group/accessibility-testing-cheat-sheet) for information on how to run these tests.
</p>
</details>
